docs/error_knowledge_base.md

```diff
@@ -418,14 +419,36 @@ See also: [Submitting a Package](how_to_add_packages.md#submitting-a-package).
 The recipe missess [short_paths](https://docs.conan.io/en/latest/reference/conanfile/attributes.html#short-paths) attribute.
 It may happen due to the very long paths within source, build or package directories during the package creating.
 Consider adding the following code:
+
 ```python
 class SomeRecipe(ConanFile):
     ...
 
     short_paths = True
 ```
 
-See also: (Maximum Path Length Limitation)[https://docs.microsoft.com/en-us/windows/win32/fileio/maximum-file-path-limitation?tabs=cmd].
+See also: [Maximum Path Length Limitation](https://docs.microsoft.com/en-us/windows/win32/fileio/maximum-file-path-limitation?tabs=cmd).
+
+#### **<a name="KB-H068">#KB-H068</a>: "TEST_TYPE MANAGEMENT"**
+
+https://github.com/conan-io/hooks/commit/ad867ad0a3656fe77ecc964e9b8f3657995f0d5d
+
+#### **<a name="KB-H069">#KB-H069</a>: "TEST PACKAGE - NO DEFAULT OPTIONS"**
+
+https://github.com/conan-io/hooks/commit/7349d593d0728ae665dcb2c92ed3038b3cbb8ee5
+
+#### **<a name="KB-H070">#KB-H070</a>: "MANDATORY SETTINGS"**
+
+https://github.com/conan-io/hooks/commit/3e7e77fa5d14ee53dc1ad6b10d85358871fdd037
+             
+#### **<a name="KB-H071">#KB-H071</a>: "INCLUDE PATH DOES NOT EXIST"**
+
+It's erroneous to leave the default `include` directory when it's no present. Consider adding:
```

Comment:
```suggestion
It's erroneous to leave the default `include` directory when it's not present. Consider adding:
```

Reply:
Not sure how much additional context is worth including. But I actually hit this problem from the consumers side.

If a consumer attempts to reference a package that has this property erroneously set, you might run into an error when using the `CMakeDeps` generator and configuring CMake:

> [CMake] CMake Error in tests/CMakeLists.txt:
> [CMake]   Imported target "foo::foo" includes non-existent
> [CMake]   path
> [CMake] 
> [CMake]     "<path/to/foo>/include"
> [CMake] 
> [CMake]   in its INTERFACE_INCLUDE_DIRECTORIES.  Possible reasons include:
> [CMake] 
> [CMake]   * The path was deleted, renamed, or moved to another location.
> [CMake] 
> [CMake]   * An install or uninstall procedure did not complete successfully.
> [CMake] 
> [CMake]   * The installation package was faulty and references files it does not
> [CMake]   provide.

Anyways, thanks for improving the documentation and giving me a reference to site :)